playwright/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/playwright:v1.20.0-focal
+
+WORKDIR /server
+RUN npm install playwright
+COPY server.js .
+
+ENV PLAYWRIGHT_PORT=4444
+ENV PLAYWRIGHT_BROWSER_TYPE=chromium
+ENV PLAYWRIGHT_HEADLESS=true
+
+EXPOSE ${PLAYWRIGHT_PORT}
+
+CMD [ "node", "server.js" ]
```

Comment:
what is the reason behind using this Dockerfile and not

```
mcr.microsoft.com/playwright:v1.20.0-focal
```

directly?

can you add a comment at the top of this as to why we prefer this way and what this does differently?

Reply:
At first, I wish to use the official image, so we don't have to maintain another image. However, after reading the [doc](https://playwright.dev/python/docs/docker), I haven't found any way to do so. The official image seems to target CI/CD platforms, where we dump source codes inside the container, install dependencies, and run tests.

Reply:
So the comment like this would be handy for others in the future to know

```
# The official image seems to target CI/CD platforms, where we dump source codes inside the container, install dependencies, and run tests.
# So we just run it like this so that we don't need to add anything more than what is necessary
```

Reply:
I think the above comment is the only change I need before merging